stdlib/Std/src/Base.luna

```diff
@@ -587,10 +587,15 @@ class List a:
         Empty: Empty
         Prepend x xs: if 0.== i then self else xs . drop i.pred
 
-    ## Returns the element at a given position in the list. Throws an error when the index does not exist.
+    ## Returns `Just` the element at a given position in the list. Returns `Nothing` when the index does not exist.
     def at n: case (self) of
-        Empty:        errorStr "List.at: index out of range"
-        Prepend x xs: if 0.== n then x else xs . at n.pred
+        Empty: Nothing
+        Prepend x xs: if 0.== n then Just x else xs . at n.pred
+
+    ## Returns the element at a given position in the list. Throws an error when the index does not exist.
+    def getAt n: case self . at n of
+        Just v: v
+        Nothing: throw "List.getAt: Index out of range."
```

Comment:
We should not hardcode function names in errors, Luna should do it automatically. It should be written as:
`Nothing: throw 'Index out of range.'`. Moreover, we should use `'` by default and prefer it over `"`.